tests/js/server/shell/shell-foxx-routing-consistency-spec.js

```diff
@@ -12,6 +12,14 @@ const SERVICE_PATH = fs.makeAbsolute(fs.join(
 ));
 const MOUNT = '/consistencytest';
 
+let options = {
+  json: true
+};
+
+if (require('internal').db._version(true)['full-version-string'].search('maintainer')) {
```

Comment:
This does not work.
`db._version(true)` returns a hierarchical object, and the attribute you are looking for is a sub-attribute inside the `details` attribute:
```{ 
  "server" : "arango", 
  "version" : "3.4.0-rc.3", 
  "license" : "enterprise", 
  "details" : { 
...
    "full-version-string" : "ArangoDB 3.4.0-rc.3 enterprise [linux] 64bit maintainer mode, using jemalloc, build heads/3.4-0-ga1af2e305f, VPack 0.1.32, RocksDB 5.16.0, ICU 58.1, V8 5.7.492.77, OpenSSL 1.1.0g  2 Nov 2017", 
...
    "maintainer-mode" : "true", 
...
  } 
}
```
Apart from that, there is an explicit `maintainer-mode` attribute, which contains a string with either "true" or "false".